.kitchen.yml

```diff
@@ -53,3 +53,16 @@ suites:
       consul:
         service_mode: cluster
         bootstrap_expect: 1
+  - name: atlas
+    run_list:
+      - recipe[consul::default]
+    attributes:
+      consul:
+        atlas_autojoin: true
+        atlas_cluster: <%= ENV.fetch('ATLAS_CLUSTER', 'example/cluster') %>
+        atlas_token: <%= ENV.fetch('ATLAS_TOKEN', 'NOT_REAL') %>
+    excludes:
+      - centos-7.0
```

Comment:
@shanesveller libarchive chooses the package name with a [switch on `platform_family?`](https://github.com/reset/libarchive-cookbook/blob/master/recipes/default.rb#L17).

I had the same problem on Arch Linux, and all it took was [an extra case](https://github.com/logankoester/libarchive-cookbook/blob/arch/recipes/default.rb#L28) to resolve it.

Guessing CentOS is in a similar position.

https://github.com/reset/libarchive-cookbook/pull/6